app/bootstrap.php

```diff
@@ -118,7 +118,7 @@
 
     // Create the 'Bolt application'
     $appClass = Application::class;
-    if ($config['application'] !== null && is_a($config['application'], Silex\Application::class, true)) {
+    if ($config['application'] !== null && is_subclass_of($config['application'], Silex\Application::class)) {
         $appClass = $config['application'];
```

Comment:
¿porque no los dos? As in, if it is a concrete `Silex\Application`, then `is_subclass_of()` will return `false`, so an `instanceof` check should follow (also to avoid deprecation notice on `is_a()`).